ext/ddtrace.c

```diff
@@ -1114,14 +1139,6 @@ static PHP_MINIT_FUNCTION(ddtrace) {
 
     ddtrace_engine_hooks_minit();
 
-#ifndef _WIN32
-    if (!get_global_DD_TRACE_SIDECAR_TRACE_SENDER()) {
-        ddtrace_coms_minit(get_global_DD_TRACE_AGENT_STACK_INITIAL_SIZE(),
-                           get_global_DD_TRACE_AGENT_MAX_PAYLOAD_SIZE(),
-                           get_global_DD_TRACE_AGENT_STACK_BACKLOG());
-    }
-#endif
-
```

Comment:
The code paths are just to complicated for my brain, but why is it safe to move this code to `dd_activate_once`?

Reply:
Background sender thread is only actually needed once spans may actually be flushed. And that never happens before dd_activate_once.